src/tools/Reporting/FailureReporting/FailureReporter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Reporting;
+using System;
+using System.IO;
+
+namespace FailureReporting
+{
+    public class FailureReporter
+    {
+        public static void Main(string[] args)
+        {
+            CreateFailureReport(args[0]);
```

Comment:
Can you add error handling and help if this is called without a parameter? Or called with a json that doesn't exist?

Reply:
What would be the desired outcome if the executable is called without a parameter (or with an invalid path)?

Reply:
I'd probably print an error to the console (so we can see it in the logs) and exit.